### PR TITLE
Extract new strings (2024-01-19)

### PR DESCRIPTION
**Pull request by mozilla-pontoon**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

File: templates/firefox-ios.xliff [hunk 1]
```diff
@@ -3816,6 +3872,14 @@ Private Tab</source>
       <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="15.0.1" build-num="15A507"/>
     </header>
     <body>
+      <trans-unit id="Addresses.Settings.EditCard.PlaceholderToggleToAllowAutofillTitle.v124" xml:space="preserve">
+        <source>Includes phone numbers and email addresses</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3876]: Should this be "Include" vs "Includes"? Unsure: for ex. following string says "Save and Fill" vs "Saves and Fills"

**Review by Delphine — COMMENTED** [review]
Just one question @adudenamedruby 
thanks!

File: templates/firefox-ios.xliff (hunk 1 above)
Comment (flodolo) [reply templates/firefox-ios.xliff:3876]: I think the string is correct, but the choice of ID is very misleading (a placeholder is the text displayed in an empty input field, this is a description or sub-text for the option).

Screenshot 2024-01-18 alle 07 46 17

**Review by Delphine — APPROVED** [review]

File: templates/firefox-ios.xliff [hunk 2]
```diff
@@ -3649,6 +3701,10 @@ Private Tab</source>
         <source>When you’re signed in and synced, you’re safer. %@ encrypts your passwords, bookmarks, and more.</source>
         <note>String used to describes the description of what Firefox is on the Sync onboarding page for current version in our Onboarding screens. Placeholder is for the app name.</note>
       </trans-unit>
+      <trans-unit id="Onboarding.Sync.Description.v123" xml:space="preserve">
+        <source>%@ encrypts your passwords, bookmarks, and more when you're synced.</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:3705]: Needs proper apostrophe

File: templates/firefox-ios.xliff [hunk 3]
```diff
@@ -4218,6 +4274,10 @@ Private Tab</source>
         <source>Privacy policy</source>
         <note>Show Firefox Browser Privacy Policy page from the Privacy section in the Shopping Experience Opt In onboarding Card (Fakespot). See https://www.mozilla.org/privacy/firefox/</note>
       </trans-unit>
+      <trans-unit id="Shopping.OptInCard.PrivacyPolicy.Button.Title.v123" xml:space="preserve">
+        <source>%@'s privacy notice</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:4278]: Again, apostrophe

File: templates/firefox-ios.xliff [hunk 4]
```diff
@@ -4230,6 +4290,10 @@ Private Tab</source>
         <source>Terms of use</source>
         <note>Show Firefox Browser Terms of Use page from the Privacy section in the Shopping Experience Opt In onboarding Card (Fakespot). See https://www.mozilla.org/privacy/firefox/</note>
       </trans-unit>
+      <trans-unit id="Shopping.OptInCard.TermsOfUse.Button.Title.v123" xml:space="preserve">
+        <source>%@'s terms of use</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:4294]: apostrophe

**Review by Delphine — CHANGES_REQUESTED** [review]
@adudenamedruby Seems like a few apostrophes fell through the cracks

File: templates/firefox-ios.xliff (hunk 2 above)
Comment (adudenamedruby) [reply templates/firefox-ios.xliff:3705]: I... can't tell? What apostrophe here?

File: templates/firefox-ios.xliff (hunk 3 above)
Comment (adudenamedruby) [reply templates/firefox-ios.xliff:4278]: I'm not sure what you mean.

File: templates/firefox-ios.xliff (hunk 4 above)
Comment (adudenamedruby) [reply templates/firefox-ios.xliff:4294]: I'm not sure what you mean.

File: templates/firefox-ios.xliff (hunk 2 above)
Comment (Delphine) [reply templates/firefox-ios.xliff:3705]: you're -> you’re 
(you can see in this PR that the linter returns errors because of the three apostrophes)

Comment (Delphine) [reply templates/firefox-ios.xliff:3705]: Documentation on typography [here](https://mozilla-l10n.github.io/documentation/misc/documentation_styleguide.html?highlight=apostrope#typography) calls out use of typographical apostrophes

Comment (flodolo) [reply templates/firefox-ios.xliff:3705]: Link to Firefox Content styleguide: https://acorn.firefox.com/latest/content/punctuation-7J3JVM4o

**Comment by adudenamedruby** [comment]
Oh, I did NOT know about this! Thanks so much. I'mma let copywriters also know!

**Comment by adudenamedruby** [comment]
Updated and fixed. :D

**Review by Delphine — APPROVED** [review]

**Pull request merged**